src/clj/rems/api/applications_v2.clj

```diff
@@ -54,7 +55,8 @@
   (-> application
       (assoc :application/modified (:event/time event))
       (assoc ::draft-answers (:application/field-values event))
-      (update :application/licenses set-accepted-licences (:application/accepted-licenses event))))
+      (update :application/licenses set-accepted-licences (:application/accepted-licenses event))
+      (update :application/accepted-licenses into (map :license/id (:application/accepted-licenses event)))))
```

Comment:
This line apparently does nothing, because the expectations in test_applications_v2.clj have not been changed except for the initial state.

Reply:
good catch! fixed in PR #1019 commit 1919135